sklearn/feature_selection/_from_model.py

```diff
@@ -135,6 +143,13 @@ class SelectFromModel(MetaEstimatorMixin, SelectorMixin, BaseEstimator):
 
         .. versionadded:: 0.24
 
+    max_features_ : int
+        Maximum number of features calculated during :term:`fit`. Only defined
+        if the ``max_features`` is not none. For details on how
+        ``max_features_`` is calculated, see ``max_features``.
```

Comment:
It's better to describe it here. (I think we can format this nicely into a sphinx list:)

```python
- If `max_features` is an int, then `max_features_ = max_features`.
- If `max_features` is a callable, then `max_features_ = max_features(X)`.
```